src/main/webapp/style.css

```diff
@@ -45,18 +99,21 @@
   color: white;
   text-decoration: none;
   padding: 14px 16px;
+  transition: 300ms cubic-bezier(0.54, 0, 0.54, 1);
 }
 
-  .nav-link:hover {
-    background-color: #111;
-  }
+.nav-link:hover {
+  background-color: #111;
+}
```

Comment:
I thought the indentation would help with visualizing the hierarchy, what do you think? It's a stylistic preference though.

Reply:
hmmm, I'm not too sure what's good practice as to CSS formatting. @dalisc any idea? we could also have our own standard.

Reply:
it's just 

class/id {
   just-indent-once
}

Reply:
there isn't nesting in css

Reply:
you can edit the indentation and merge this in!

Reply:
yup, I removed the indentation in the previous commits.